modules/cluster-management/src/test/java/org/apache/ignite/internal/cluster/management/raft/AbstractClusterStateStorageTest.java

```diff
@@ -110,6 +110,50 @@ void testPutReplace() {
         assertThat(storage.get(key), is(equalTo(value2)));
     }
 
+    @Test
+    void testReplaceAll() {
```

Comment:
This test actually contains 3 test cases for 3 properties of the tested method. How about breaking the test method to 3, one per test case?

Reply:
we have a lot of similar situations in other tests. In this case they are very small, do you think we will benefit from that?